internal/utils/utils.go

```diff
@@ -156,3 +157,65 @@ func Start_prometheus_server() {
 		_ = http.ListenAndServe(fmt.Sprintf(":%s", cfg.PrometheusPort), nil)
 	}
 }
+
+func ConvertMemoryFromBytesToMiB(jsonData datatypes.JSON) map[string]interface{} {
```

Comment:
Please shift this function to `utils.go` which is under the `api` directory.

Reply:
Done